go.mod

```diff
@@ -2,14 +2,16 @@ module github.com/ethereum/go-ethereum
 
 go 1.19
 
+replace github.com/holiman/billy => github.com/karalabe/billy v0.0.0-20230331141121-732fa9109cff
```

Comment:
Should we upstream the changes to `holiman/billy` and revert this change?

Reply:
Ah yeah, the go.mod file should have been left out of this PR, thx.